src/Hamiltonians.jl

```diff
@@ -1387,6 +1411,18 @@ numberoccupiedsites(b::BoseFS) = numberoccupiedsites(b.bs)
 numberoccupiedsites(a::BSAdd64) = numberoccupiedsites(a.add)
 numberoccupiedsites(a::BSAdd128) = numberoccupiedsites(a.add)
 
+function numberoccupiedsites(b::BoseFS2C{NA,NB,M,AA,AB}) where {NA,NB,M,AA,AB}
```

Comment:
Is this method needed? If not, please remove!

Reply:
Removed.